Mathlib/Topology/Sets/Compacts.lean

```diff
@@ -445,70 +445,73 @@ theorem coe_mk (s : Compacts α) (h) : (mk s h : Set α) = s :=
 instance : Sup (CompactOpens α) :=
   ⟨fun s t => ⟨s.toCompacts ⊔ t.toCompacts, s.isOpen.union t.isOpen⟩⟩
 
-instance [QuasiSeparatedSpace α] : Inf (CompactOpens α) :=
-  ⟨fun U V =>
-    ⟨⟨(U : Set α) ∩ (V : Set α),
-        QuasiSeparatedSpace.inter_isCompact U.1.1 V.1.1 U.2 U.1.2 V.2 V.1.2⟩,
-      U.2.inter V.2⟩⟩
+instance : Bot (CompactOpens α) where bot := ⟨⊥, isOpen_empty⟩
```

Comment:
How about this file? As far as I can see, these changes are basically orthogonal to your refactor. Or am I missing something?

Reply:
Almost orthogonal. I need the `himp` field now

Reply:
I put these changes in #15005 as well